spec.bs

```diff
@@ -3003,6 +3014,49 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|config|)</dfn> meth
 
 </div>
 
+<div algorithm>
+  To <dfn>parse and verify ad auction config</dfn> given an {{AdAuctionDataConfig}} |configIDL|:
+
+  1. Let |config| be a new [=auction data config=] with the following [=struct/items=]:
+    : [=auction data config/seller=]
+    :: The result of running [=parsing an https origin=] on
+       |configIDL|["{{AdAuctionDataConfig/seller}}"]
+    : [=auction data config/coordinator=]
+    :: The result of running [=parsing an https origin=] on
+       |configIDL|["{{AdAuctionDataConfig/coordinatorOrigin}}"]
+    : [=auction data config/request size=]
+    :: |configIDL|["{{AdAuctionDataConfig/requestSize}}"]

```

Comment:
this field may not exist. Need to check whether it exists before getting its value.
```suggestion
    :: |configIDL|["{{AdAuctionDataConfig/requestSize}}"] if it [=map/exists=], null otherwise
```